components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4927,6 +4947,38 @@ where
             self.register_reactivate_memory_lock_tick();
         }
     }
+
+    fn on_report_region_buckets_tick(&mut self) {
```

Comment:
Region counts should be small when dynamic region is enabled, so we don't care about its report frequency?

Reply:
Yes, it is supposed to have little impact.